.github/.check_scopes.lua

```diff
@@ -0,0 +1,38 @@
+function extractScopesFromFile(filename)
+    local scopes = {}
+    local file = io.open(filename, "r")
```

Comment:
By default, `io.open` opens the file in read mode. We could, therefore, omit the `"r"`.